app/main/posts/modify/post-data-editor.html

```diff
@@ -106,7 +106,7 @@
         </post-tabs>
       <div class="toolbar toolbar-secondary">
         <div class="button-group">
-          <button class="button-flat" ng-click="leavePost()" translate>app.cancel</button>
+          <button class="button-flat" ui-sref="list.data.detail({postId: post.id})" translate>app.cancel</button>
```

Comment:
I see you've changed this handling to watch transitions, instead of directly calling `leavePost` to check for changes. Is the main reason because now you can watch for *all* state changes, not just the button press?

Reply:
I changed this because it represents what you are trying to do in a better way than leave post did and it's more consistent to what you have to do everywhere in the cancel button. For instance, you now can use the same ui-sref in post-toolbar button and it will behave EXACTLY the same as this cancel button, versus being "kind of the same but it could also be completely different who knows" which was what we had =)